lib/internal/test_runner/test.js

```diff
@@ -584,10 +614,12 @@ class Test extends AsyncResource {
     this.finished = true;
   }
 
-  report() {
+  get #duration() {
```

Comment:
why not make this a function and call it with `()`? making it a getter is just slightly slower and less explicit for a 2-character gain at callsites inside the same file

Reply:
I was not aware of getters being slower, can you refer me to where I can read about that? Interesting, thanks!

Reply:
i think the difference is negligible in modern v8, to be fair, but i find that getters make code less readable - `this.#duration` isn't a meaningful improvement over `this.#duration()`, but in the former, it's not obvious it's calling a function, so it can be much more confusing to understand and debug